nuget/lib/BlingFireUtils.cs

```diff
@@ -1,26 +1,40 @@
+// <copyright file="BlingFireUtils.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+//     Licensed under the MIT License.
+// </copyright>
+
 using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Runtime.InteropServices;
+using System.Text;
 
 /// <summary>
 /// This is C# interface for blingfiretokdll.dll
 /// </summary>
 namespace BlingFire
 {
-    public static class BlingFireUtils
+    public static class BlingFire
     {
         private const string BlingFireTokDllName = "blingfiretokdll";
 
         [DllImport(BlingFireTokDllName)]
-        public static extern Int32 GetBlingFireTokVersion();
+        static extern Int32 GetBlingFireTokVersion();
+
+        public static Int32 GetBlingFireTokVersion(int fake = 0)
+        {
+            return GetBlingFireTokVersion();
+        }
+        
 
         [DllImport(BlingFireTokDllName)]
-        public static extern UInt64 LoadModel(byte[] modelName);
+        static extern UInt64 LoadModel(byte[] modelName);
 
-        public static UInt64 LoadModel(string modelName)

```

Comment:
the public LoadModel should return long the types between LoadModel/FreeModel should match

Reply:
reverted

Reply:
I feel it is best to not make this change at this moment.